postgresql/migrations/000014_lsps2_buy.up.sql

```diff
@@ -15,3 +15,18 @@ CREATE TABLE lsps2.buy_registrations (
 );
 CREATE UNIQUE INDEX idx_lsps2_buy_registrations_scid ON lsps2.buy_registrations (scid);
 CREATE INDEX idx_lsps2_buy_registrations_valid_until ON lsps2.buy_registrations (params_valid_until);
+
+CREATE TABLE lsps2.bought_channels (
+	id bigserial PRIMARY KEY,
```

Comment:
It's perhaps better to use uuid7 instead (there is a pg extension to generate them but it's better imo to use a go generator like https://github.com/GoWebProd/uuid7)